src/main/kotlin/com/exactpro/th2/rptdataprovider/Context.kt

```diff
@@ -58,7 +61,10 @@ class Context(
 
     val jacksonMapper: ObjectMapper = jacksonObjectMapper()
         .enable(DeserializationFeature.FAIL_ON_READING_DUP_TREE_KEY)
-        .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES),
+        .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
+        .registerModule(SimpleModule("backward_compatibility").apply {
+            addSerializer(Instant::class.java, InstantBackwardCompatibilitySerializer)
+        }),
```

Comment:
Or I guess we could add dependency on `com.fasterxml.jackson.datatype:jackson-datatype-jsr310`. Not sure if it has to be registered explicitly though

Reply:
> Not sure if it has to be registered explicitly though

Looks like it is - by adding `JavaTimeModule` module

Reply:
We could add it but it does not serialize the value as it is expected by UI. That is why the custom serializer was created